test/gateway/schema.js

```diff
@@ -2385,7 +2385,7 @@ test('Update the schema', async (t) => {
   gateway.graphql.gateway.serviceMap.working.setSchema(fullSchema)
   const newSchema = await gateway.graphql.gateway.refresh()
 
-  gateway.graphql.replaceSchema(newSchema)
+  gateway.graphql.replaceSchema(newSchema.schema)
```

Comment:
why this change?

Reply:
the `gateway.refresh()` function now returns an object `{ schema, failedServices }` instead of only the `schema` which is why I had to change this.

Reply:
Unfortunately this would be a semver-major change that I would like to avoid. Can you rearrange the API differently?